src/task1/Main.java

```diff
@@ -0,0 +1,50 @@
+package task1;
+
+public class Main {
+    private static User[] users = {new Human("Ban", 50, 2),
+            new Cat("Kasper", 100, 3),
+            new Robot("R2D2", 150, 1),
+            new Robot("Optimus Prime", 500, 6)};
+    private static Obstacle[] obstacles = {new RaceTrack(48),
+            new Wall(1),
+            new RaceTrack(80),
+            new Wall(2),
+            new RaceTrack(100),
+            new Wall(5)};
+
+    public static void main(String[] args) {
+        startChallenge();
+        System.out.println();
+        printWinners();
+
+    }
+
+    public static void startChallenge() {
```

Comment:
Would be better to create a new class `Competition` and add the method `start` that will contain this logic